packages/server/test/unit/cloud/api/api_spec.js

```diff
@@ -323,12 +325,75 @@ describe('lib/cloud/api', () => {
       })
     })
 
-    it('sets timeout to 60 seconds', () => {
+    it('sets timeout to 60 seconds when no CYPRESS_INITIAL_PREFLIGHT_TIMEOUT env is set', () => {
       sinon.stub(api.rp, 'post').resolves({})
 
       return api.sendPreflight({})
       .then(() => {
-        expect(api.rp.post).to.be.calledWithMatch({ timeout: 60000 })
+        expect(api.rp.post).to.be.calledWithMatch({ timeout: 5000 })
+      })
+    })
+
+    describe('when CYPRESS_INITIAL_PREFLIGHT_TIMEOUT env is set to a negative number', () => {
+      const configuredTimeout = -1
+      let prevEnv
+
+      beforeEach(() => {
+        prevEnv = process.env.CYPRESS_INITIAL_PREFLIGHT_TIMEOUT
+        process.env.CYPRESS_INITIAL_PREFLIGHT_TIMEOUT = configuredTimeout
+      })
+
+      afterEach(() => {
+        process.env.CYPRESS_INITIAL_PREFLIGHT_TIMEOUT = prevEnv
+      })
+
+      it('skips the no-agent preflight request', () => {
+        preflightNock(API_PROD_PROXY_BASEURL)
+        .replyWithError('should not be called')
```

Comment:
I'm missing where this logic is implemented..I thought it would default to 5000ms